dist/Dockerfile.deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM local/cincinnati-build:latest AS builder
+FROM quay.io/app-sre/cincinnati:base AS builder
```

Comment:
If we're going to pick a dummy name for the build container, can we use `quay.io/app-sre/cincinnati:builder`?  I think of `base` as the next image into which the built binaries are copied.

Reply:
Makes sense, done

Reply:
Looks like the name here is still `base` as opposed to @wking's suggestion `builder`.